### PR TITLE
Appdome build 2secure android 1.0.4

### PR DESCRIPTION
**Pull request by elish266**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Review by godrei — CHANGES_REQUESTED** [review]
Hi @elish266,
thanks for this PR. 
I found some room for improvement, please check my comments. Here is the guide on how issues should be fixed in a StepLib PR: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/sharing-steps-with-all-bitrise-users.html#fixing-issues-in-a-steplib-pull-request

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- code-sign
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- app_location: null
+  opts:
+    is_required: true
+    summary: URL to app file
+    title: App file URL
```
Comment (koral--) [inline steps/appdome-build-2secure-android/1.0.4/step.yml:26]: It does not make sense to add summary almost equal to title.

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  git: https://github.com/Appdome/bitrise-step-appdome-build-2secure-android.git
+  commit: 0724cf2d140cdfb6fe2b281984fdd1810615c3b4
+project_type_tags:
+- android
+type_tags:
+- build
+- code-sign
+toolkit:
+  bash:
+    entry_file: step.sh
```
Comment (koral--) [inline steps/appdome-build-2secure-android/1.0.4/step.yml:20]: The script seems to use curl which is not listed as a dependency here.

Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:20]: will be fixed in next PR for release 1.0.5.

File: steps/appdome-build-2secure-android/1.0.4/step.yml (hunk 1 above)
Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:26]: will be fixed in the next PR for release 1.0.5.

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    is_required: true
+    title: Signing Method
+    value_options:
+    - On-Appdome
+    - Private-Signing
+    - Auto-Dev-Signing
+  sign_method: On-Appdome
+- gp_signing: "true"
+  opts:
+    description: Sign the app for Google Play? If 'true', requires $SIGN_FINGERPRINT
```
[the remark this answers is not in the document]
Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:46]: SIGN_FINGERPRINT is stored as a secret, as we prefer not to include it in the user input as it is not always required. We have six signing options while it is required only in three of them. Not having SING_FINGERPRINT in the user input screen is our UI design decision.

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+project_type_tags:
+- android
+type_tags:
+- build
+- code-sign
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- app_location: null
```
[the remark this answers is not in the document]
Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:22]: will be addressed in next PR for release 1.0.5.

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+title: Appdome-Build-2Secure for Android
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, building and signing mobile apps using Appdome's API. For details see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
+website: https://github.com/Appdome/bitrise_build-2secure-android
+source_code_url: https://github.com/Appdome/bitrise_build-2secure-android
+support_url: https://github.com/Appdome/bitrise_build-2secure-android/issues
```
[the remark this answers is not in the document]
Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:8]: will be addressed in next PR for release 1.0.5.

File: steps/appdome-build-2secure-android/1.0.4/step.yml [hunk 6]
```diff
@@ -0,0 +1,52 @@
+title: Appdome-Build-2Secure for Android
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, building and signing mobile apps using Appdome's API. For details see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
```
[the remark this answers is not in the document]
Comment (elish266) [reply steps/appdome-build-2secure-android/1.0.4/step.yml:5]: Known issue. This URL will be available after official release of the step.

**Pull request closed without merging**